const hasError = (resArray) => resArray.some((res) => res.error);

module.exports = hasError;
